tests/integration/azure/test_credentials.py

```diff
@@ -168,17 +178,36 @@ def test_spn_migration(ws, extract_test_info, run_migration, read_only):
         assert match is not None, "PATH_EXISTS validation should fail"
 
 
-@pytest.mark.skip(reason="TODO: Let migration create storage credentials.")
 @retried(on=[InternalError], timeout=timedelta(minutes=2))
-def test_spn_migration_access_connector_created(ws, extract_test_info, run_migration):
+def test_spn_migration_access_connector_created(
+    az_cli_ctx, env_or_skip, extract_test_info, run_migration, product_info, make_random
+):
     """Storage credentials should be created for the access connectors."""
-    storage_credentials = [sc for sc in ws.storage_credentials.list() if sc.name.startswith("ac-")]
-    assert len(storage_credentials) == 0
+    # Mocking in an integration test because Azure resource can not be created
+    resource_permissions = create_autospec(AzureResourcePermissions)
+
+    # TODO: Remove the replace!
+    access_connector_id = AzureResource(env_or_skip("TEST_ACCESS_CONNECTOR").replace("-external", ""))
```

Comment:
@nfx : This integration test works with non-external access connector locally.

Reply:
yep, let's look tomorrow.

Reply:
requires infra change, we use non-external access connector for the time being. I will remove it when the change is implemented